src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+import java.io.*;
```

Comment:
I agree with Btaykb, shouldn't all packages imported from java.io be written out to follow the coding standards?